src/ensembl/src/content/app/browser/browserState.ts

```diff
@@ -23,15 +23,28 @@ const activeEnsObjectIds = browserStorageService.getActiveEnsObjectIds();
 const trackStates = browserStorageService.getTrackStates();
 const chrLocations = browserStorageService.getChrLocation();
 
+export enum BrowserNavIcons {
```

Comment:
If it's an enum, it's describing a single thing at a time; therefore it should be singular rather than plural. Icon, not Icons. Maybe even Action rather than Icon (icon is too much of a concrete UI image thing).

Reply:
That's done